### PR TITLE
Governance updates and clarifications

### PR DESCRIPTION
**Pull request by OriolAbril**
----
:books: Documentation preview :books:: https://arviz-project--41.org.readthedocs.build/en/41/

### CODE REVIEW

File: governance/community_architecture.md [hunk 1]
```diff
@@ -74,6 +77,11 @@ their contributions and their expert knowledge of the Project Software and
 Services that Council Members will provide useful guidance, both technical and
 in terms of project direction, to potentially less experienced contributors.
 
+The Council will have between 4 and 7 members.
```
Comment (OriolAbril) [inline governance/community_architecture.md:80]: We had 10 people max, which now in hindsight seems a bit too much.

File: governance/community_architecture.md [hunk 2]
```diff
@@ -74,6 +77,11 @@ their contributions and their expert knowledge of the Project Software and
...
+No more than 2 Council Members can report to one person or company
+(including Institutional Partners) through employment or contracting work
+(including the reportee, i.e. the reportee + 1 is the max).
```
Comment (OriolAbril) [inline governance/community_architecture.md:83]: We didn't have anything like this, but the contract we signed with numfocus did, so I added it

File: governance/processes.md [hunk 3]
```diff
@@ -113,15 +116,51 @@ issue, but must recuse themselves from voting on the issue.
...
 * In the event that either not enough or too many candidates were confirmed,
   candidates are ranked by interpreting yes=+1, neutral=0 and no=-1. If too
-  many candidates were confirmed, the 10 candidates with higher rank are
+  many candidates were confirmed, the 7 candidates with higher rank are
   elected. If not enough candidates were chosen, the 4 candidates with higher
-  rank are elected.
+  rank are elected. If affiliation conditions were not met, multiple potential
+  councils will be constructed with candidates whose rank is `>i` for all possible
+  values of `i`. If any of those potential councils matches the conditions,
+  the one with smaller `i` will be chosen.
```
Comment (OriolAbril) [inline governance/processes.md:132]: Tried to add the extra behaviour for when the # or candidates is in the accepted range but still conditions aren't met. Let me know what you think.

File: governance/processes.md [hunk 4]
```diff
@@ -168,10 +207,3 @@ issue, but must recuse themselves from voting on the issue.
 Core contributors can also voluntarily leave the project by notifying the community through a public means or by notifying the entire council.
 
 Unless they request otherwise, they will be listed on {ref}`emeritus` page.
-
-## Voting Criteria For Future Elections
-Voting for first election is restricted to establish stable governance, and to defer major decision to elected leaders
-* For the first election only the people registered following the guidelines in `elections/ArviZ_2020.md` can vote
-* In the first year, the council must determine voting eligibility for future elections between two criteria:
-  * Core contributors
-  * The contributing community at large
```
Comment (OriolAbril) [inline governance/processes.md:177]: I have deleted this part, however, we might want to reconsider (or consider as we didn't really discuss this when adding the recurring contributor figure). Do we want recurrent contributors to vote on the election? That would mean they can't be candidates nor have commit rights but they would be able to vote in the elections in addition to being in slack

**Comment by aloctavodia** [comment]
LGTM!

**Pull request merged**